data/brands/amenity/pharmacy.json

```diff
@@ -2476,6 +2476,42 @@
         "name": "Watsons"
       }
     },
+    {
+      "displayName": "屈臣氏",
+      "locationSet": {"include": ["cn"], "exclude": ["hk", "mo"]},
+      "tags": {
```

Comment:
Watsons had been in `shop/chemist`

https://github.com/osmlab/name-suggestion-index/blob/de589448d288c4bf880afc780a27c3567cec5619/data/brands/shop/chemist.json#L603-L621

Reply:
Thanks for your finding. Let's use `shop/chemist`.

Reply:
Should we move Watsons into `shop/chemist` too?

Reply:
> Should we move Watsons into `shop/chemist` too?

Yes, and add a matchTag so that pharmacy one will be redirected.

Reply:
> > Should we move Watsons into `shop/chemist` too?
> 
> Yes, and add a `matchTag` so that pharmacy one will be redirected.

Added.

Reply:
Besides, this is a Hong Kong brand, so can you decide that whether waston/wastons is correct?

Reply:
> Besides, this is a Hong Kong brand, so can you decide that whether waston/wastons is correct?

Watsons is the correct name. Updated.